css/components.css

```diff
@@ -0,0 +1,517 @@
+/* button Started */
+.avtr,.avtr-clr,.sm-avtr-clr,.avtr-sq,.sm-avtr-sq{
+    position: relative;
+    padding: 4px;
+}
+.avtr-clr > h4,.sm-avtr-clr > h4,.avtr-sq> h4{
+    border-radius: 50%;
+    background: var(--primary-light);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 2.5rem;
+    height: 2.5rem;
+    color: var(--inner-text);
+}
+.avtr-sq>h4{
+    border-radius: 0.25rem;
+}
+.sm-avtr-clr > h4{
+    width: 2rem;
+    height: 2rem;
+}
+.avtr-clr > img , .sm-avtr-clr > img , .avtr-sq > img,.sm-avtr-sq > img {
+    object-fit: cover;
+    border-radius: 50%;
+}
+
+.avtr-sq > img{
+    border-radius: 0.375rem;
+}
+.sm-avtr-sq > img{
+    border-radius: 0.25rem;
+}
+.avtr-clr > img, .avtr-sq > img {
+    width: 2.5rem;
+    height: 2.5rem;
+}
+.sm-avtr-clr > img, .sm-avtr-sq > img{
+    width: 32px;
+    height: 32px;
+}
+.bdge{
+    box-sizing: border-box;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.125rem 0.25rem ;
+    bottom: 0.0625rem ;
+    left: 65% ;
+    min-width: 1rem;
+    height: 1rem;
+    font-weight: 700;
+    font-size: 0.65rem;
+    text-align: center;
+    border-radius: 0.5rem;
+    color: var(--inner-text);
+    background: var(--red-dark);
+    box-shadow:0 0 0 0.24em var(--background);
+}
+
+/* Buttons */
+button ,.link-btn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font: 500 1rem/1.18em 'Lato', sans-serif;
```

Comment:
## Standard Unit values
1. Keep unit values standard across files, if you are using `rem`  use that only everywhere.